core/libdeno/modules.cc

```diff
@@ -147,7 +147,6 @@ void deno_mod_evaluate(Deno* d_, void* user_data, deno_mod id) {
   if (status == Module::kInstantiated) {
     bool ok = !module->Evaluate(context).IsEmpty();
     status = module->GetStatus();  // Update status after evaluating.
-    CHECK_IMPLIES(ok, status == Module::kEvaluated);
```

Comment:
That looks wrong. What is the status here, then?

Reply:
I believe sometimes it's now kEvaluating

Reply:
Here are a subset of the tests which have kEvaluating at this check

```
test error_001 ... FAILED
test error_002 ... FAILED
test error_004_missing_module ... ok
test error_005_missing_dynamic_import ... ok
test error_006_import_ext_failure ... ok
test error_003_typescript ... ok
test error_008_checkjs ... FAILED
test error_011_bad_module_specifier ... ok
test error_012_bad_dynamic_import_specifier ... ok
test error_013_missing_script ... ok
test error_014_catch_dynamic_import_error ... FAILED
test error_015_dynamic_import_permissions ... ok
test error_003_typescript2 ... ok
test error_016_dynamic_import_permissions2 ... ok
test error_syntax ... ok
test error_type_definitions ... ok
test error_007_any ... FAILED
test error_stack ... FAILED
```

Reply:
Actually I'm wrong, it's not kEvaluating but kErrored sometimes when `ok`. I've updated the assert.